### PR TITLE
[Reviewer: Matt] Simpler orchestration of Cacti and switch to SNMP

### PR DESCRIPTION
**Pull request by rkd-msw**
Matt,

I've:
* created SNMP Cacti templates for the Bono connection count and Sprout and Bono latency
* created Cacti host templates for Sprout, Bono and SIPp
* changed the Cacti recipe so that it
  * loads in a SQL database dump which is preconfigured with the settings from http://clearwater.readthedocs.org/en/latest/Cacti/index.html (e.g. setting the 'spine' poller)
  * loads the host templates
* added a `knife cacti update -E ENV` command, which runs through all your Sprout/Bono/SIPp nodes and adds thm all to Cacti

Can you review? Reviewing the templates might be better done by me spinning up a Cacti node and pointing you at it - let me know if you want that.

### CODE REVIEW

File: plugins/knife/knife-cacti-update.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        find_nodes(roles: "clearwater-infrastructure", role: "sprout").each do |node|
+          run_command(options[:cloud], "chef_environment:#{env} AND name:#{cacti.name}", "sudo bash /usr/share/clearwater/cacti/add_device.sh #{node.cloud.local_ipv4} #{node.name} Sprout")
+        end
+
+        find_nodes(roles: "clearwater-infrastructure", role: "sipp").each do |node|
+          run_command(options[:cloud], "chef_environment:#{env} AND name:#{cacti.name}", "sudo bash /usr/share/clearwater/cacti/add_device.sh #{node.cloud.local_ipv4} #{node.name} SIPp")
+        end
+
+        end
```
Comment (mirw) [inline plugins/knife/knife-cacti-update.rb]: Either your indentation is out, or you've got a duplicate end here - it looks like the latter.  Does this work?

File: cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_bono.xml [hunk 2]
```diff
@@ -0,0 +1,1856 @@
+<cacti>
```
Comment (mirw) [inline cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_bono.xml]: How do we keep these templates in sync with those referenced by http://clearwater.readthedocs.org/en/latest/Cacti/?

Comment (mirw) [inline cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_bono.xml]: I haven't reviewed these templates - presumably they work, though?

**Comment by mirw** [comment]
Looks good.  A few comments in line.

File: cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_bono.xml (hunk 2 above)
Comment (rkd-msw) [inline cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_bono.xml]: I was planning, as part of my doc updates, to make http://clearwater.readthedocs.org/en/latest/Cacti/ point into this repository at these templates (and remove the current ones).

Comment (mirw) [inline cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_bono.xml]: Fine.

**Pull request merged**